recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: spyder
+  version: 2.3.9
+
+source:
+  fn:  spyder-2.3.9.zip
+  url: https://pypi.python.org/packages/fb/37/09b789dbe321894afe1657f11f0026a5ea79987f1c30176b92ee648b3faa/spyder-2.3.9.zip
+  md5: dd01e07a77123c128ff79ba57b97c1d7
+
+build:
+  number: 0
+  osx_is_app: True
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pyqt
+    - rope
+    - pyflakes
+    - jedi >=0.8
+    - qtconsole
+    - nbconvert
+    - pygments
+    - sphinx
+    - pep8
+    - psutil
+    - python.app      [osx]
+
+app:
+  entry: spyder
+  icon: logo.png
+  summary: Scientific Python Development Environment
+  type: desk
+
+test:
+  commands:
+    - spyder -h
+  imports:
+    - spyderlib
+
+about:
+  home: https://github.com/spyder-ide/spyder
+  license: MIT
+  license_file: LICENSE
+  summary: Scientific PYthon Development EnviRonment
+  description: |
+    Spyder is a powerful interactive development environment for the Python
+    language with advanced editing, interactive testing, debugging and
+    introspection features. Additionally, Spyder is a numerical computing
+    environment thanks to the support of IPython and popular Python libraries
+    such as NumPy, SciPy, or matplotlib.
+  doc_url: http://pythonhosted.org/spyder/
+  dev_url: https://github.com/spyder-ide/spyder
```

Comment:
There is a mistake here

home should be: http://spyder-ide.org

The home web page is under construction so it is pointing to the github repo in the meantime